lib/utils/helper.dart

```diff
@@ -164,3 +166,43 @@ Future updateAndroidWidgets() async {
     await HomeWidget.updateWidget(name: name, androidName: name);
   }
 }
+
+Future<bool> requestLocationPermission() async {
+  // Request the location permission.
+  var status = await Permission.location.request();
+
+  // Check if permission was granted.
+  if (status == PermissionStatus.granted) {
+    return true;
+  }
+
+  return false;
+}
+
+Future<bool> requestNotificationsPermission() async {
+  // Request the location permission.
+  var status = await Permission.notification.request();
+
+  // Check if permission was granted.
+  if (status == PermissionStatus.granted) {
+    return true;
+  }
+
+  return false;
+}
```

Comment:
These 2 can be removed, not used anymore

Reply:
for feature usage maybe?

Reply:
I'm not sure since the other covers it, but sure.